schemas/definition-1.0.json

```diff
@@ -32,6 +32,8 @@
     "type": {
       "enum": [
         "npm",
+        "conda",
+        "condasrc",
```

Comment:
```suggestion
        "condasource",
```

Reply:
Thanks, Good spot, Lisa!

Reply:
Actually, it's intentional, to be consistent with their naming convention, i.e. debsrc.
https://github.com/clearlydefined/crawler/blob/b3bf591f15c12643dc9536edc94c66dace637c4f/config/cdConfig.js#L57